libs/designsystem/item-sliding/src/item-sliding.component.spec.ts

```diff
@@ -140,6 +140,13 @@ describe('ItemSlidingComponent', () => {
         ionItemOptionElements[idx].click();
         expect(onSelectedSpy).toHaveBeenCalled();
       });
+
+      it('should call close', () => {
+        const idx = 0;
+        const onSelectedSpy = spyOn(spectator.component.itemSliding, 'close');
+        ionItemOptionElements[idx].click();
+        expect(onSelectedSpy).toHaveBeenCalled();
```

Comment:
```suggestion
     expect(closeSpy).toHaveBeenCalled(); 
     expect(closeSpy).toHaveBeenCalledTimes(1);
```

Reply:
You could consider adding an addition `expect` that confirms that the `close` method is only called once.